spring-core/src/test/java/org/springframework/util/SocketUtilsTests.java

```diff
@@ -53,6 +53,13 @@ public void findAvailableTcpPort() {
 		assertPortInRange(port, PORT_RANGE_MIN, PORT_RANGE_MAX);
 	}
 
+	@Test
+	public void findAvailableTcpPortWithMinPortEqualsMaxPort() {
+		int minMaxPort = 20000;
+		int port = SocketUtils.findAvailableTcpPort(minMaxPort, minMaxPort);
+		assertEquals(minMaxPort, port);
```

Comment:
If port 20,000 is not available, this test is guaranteed to fail.

Thus, we should not introduce a test that is this fragile.

Please make the test robust.